CRM/Core/BAO/CustomGroup.php

```diff
@@ -358,10 +358,17 @@ public static function &getTree(
     if ($entityID) {
       $entityID = CRM_Utils_Type::escape($entityID, 'Integer');
     }
+    if (!is_array($subTypes)) {
+      if (empty($subTypes)) {
+        $subTypes = array();
+      }
+      else {
+        $subTypes = explode(',', $subTypes);
```

Comment:
Either ways, it seems like we need to explode on CRM_Core_DAO::VALUE_SEPARATOR here as the validateSubTypeByEntity function will not work if passed a VALUE_SEPARATOR-style list (it validates a single subType).